src/edu/csus/ecs/pc2/ui/RunInputValidatorsPane.java

```diff
@@ -235,23 +236,54 @@ protected void populateGUI() {
         
     }
 
-    
+    /**
+     * This method returns a Vector containing one {@link ProblemInputValidationResults} element for each problem in the 
+     * specified array of {@link Problem}s which has an Input Validator associated with the problem and for which the associated
+     * Input Validator has been run (executed).  Each {@link ProblemInputValidationResults} element contains the Input Validation
+     * results for the Input Validator (either a Custom Input Validator or the VIVA Input Validator) currently associated with 
+     * the corresponding Problem (if any).  If a Problem has no associated Input Validator then the returned Vector will contain
+     * no {@link ProblemInputValidationResults} entry for that problem.  (Note that each {@link ProblemInputValidationResults} 
+     * object contains a field identifying the Problem to which it applies.)
+     * 
+     * @param probs an Array of Problems
+     * 
+     * @return a Vector containing a {@link ProblemInputValidationResults} element for each Problem which has an Input Validator 
+     *          associated with the Problem.
+     *          
+     */
     private Vector<ProblemInputValidationResults> getInputValidationResultsTableData(Problem [] probs) {
         
         Vector<ProblemInputValidationResults> tableData = new Vector<ProblemInputValidationResults> ();
         
         for (int prob=0; prob<probs.length; prob++) {
-            if (probs[prob].isProblemHasInputValidator()) {
-                
-                ProblemInputValidationResults result = new ProblemInputValidationResults(probs[prob]);
-                
-                // add each result for the current problem to the current row
-                for (InputValidationResult res : probs[prob].getInputValidationResults()) {
-
-                    result.addResult(res); // do we need to clone the InputValidationResult?
+            //check if the problem has a Custom Input Validator
+            if (probs[prob].getCurrentInputValidatorType()==INPUT_VALIDATOR_TYPE.CUSTOM) {

```

Comment:
To each their own style; your approach would require executing an extra runtime statement to assign the value of probs[prob] to the variable "problem" so that the variable could then be used 'more clearly' (in your opinion).  I'm not convinced that's a worthwhile tradeoff -- but again, to each their own style.

I'm assuming you don't think this rises to the level of rejecting the PR because of it.  If that's not the case, feel free "unresolve" the conversation and/or add a new Issue for it.  (But I don't see anyone getting around to spending their valuable time on fixing such a trivial item...)